contracts/test/vault/oethb-vault.base.fork-test.js

```diff
@@ -113,20 +113,101 @@ describe("ForkTest: OETHb Vault", function () {
   });
 
   describe("Async withdrawals", function () {
-    it("Should be disabled", async () => {
-      const { oethbVault, nick } = fixture;
+    it("Should allow 1:1 async withdrawals", async () => {
+      const { rafael, oethbVault } = fixture;
+
+      const delayPeriod = await oethbVault.withdrawalClaimDelay();
+
+      if (delayPeriod == 0) {
+        // Skip when disabled
+        return;
```

Comment:
I think we should have a fixture here that sets claim delays to some constant, say 10 minutes. If we don't it could happen that: 
- we disable async withdrawals for whatever reason on the mainnet
- introduce a code change that breaks the functionality
- re-enable on the mainnet and we wouldn't detect it